libs/coin-modules/coin-bitcoin/src/signOperation.ts

```diff
@@ -92,8 +89,8 @@ export const buildSignOperation =
             lockTime,
             sigHashType,
             segwit,
-            hasTimestamp,
-            initialTimestamp,
+            hasTimestamp: false,
+            initialTimestamp: undefined,
```

Comment:
does `initialTimestamp` ever get set anywhere now?

Reply:
No it doesn't
However I looked into when it was called and it's a bit of a rabbit hole so I just keep the changes limited

Reply:
Yeah it could be a separate issue then, you're right

Reply:
we can get rid of them. but of course, we can do it in another task if you want.